.travis.yml

```diff
@@ -18,9 +18,11 @@ install:
     - export DEV_LIB_PATH=dev-lib
     - if [ ! -e "$DEV_LIB_PATH" ] && [ -L .travis.yml ]; then export DEV_LIB_PATH=$( dirname $( readlink .travis.yml ) ); fi
     - if [ ! -e "$DEV_LIB_PATH" ]; then git clone https://github.com/xwp/wp-dev-lib.git $DEV_LIB_PATH; fi
+    - npm install
```

Comment:
Oh! Thanks, I didn't know about wp-dev-lib. I clearly didn't set up my fork of this repo correctly with the submodules.